web/src/questions-utils.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import Popup from "./Popup";
+import { partition } from "./utils";
+
+/**
+ * Returns given text capitalized
+ *
+ * @param {String} text - string to be capitalized
+ * @returns {String} capitalized text
+ */
+const label = text => `${text[0].toUpperCase()}${text.slice(1)}`;
```

Comment:
Not needed by now, but this should be improved (or changed) to work with i18n too. Not to mention, that a "notNow" option will be transformed to "NotNow" instead of "Not now".

Reply:
Yes, IMHO this deserves a TODO comment.